+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "1286"
    creation_ts: 2012-01-25 11:33:00 +0100
    short_desc: test data for datatype_sens is not present
    delta_ts: 2014-03-12 12:21:35 +0100
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: PC
    op_sys: Windows
    bug_status: CLOSED
    resolution: FIXED
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P3
    bug_severity: trivial
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: jorn
      +@name: Jörn M. Horschig
    assigned_to:
      +content: jorn
      +@name: Jörn M. Horschig
    cc:
      - j.schoffelen
      - r.oostenveld
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "4927"
        comment_count: "0"
        who:
          +content: jorn
          +@name: Jörn M. Horschig
        bug_when: 2012-01-25 11:33:41 +0100
        thetext: |-
          ??? Error using ==> ft_read_sens at 65
          file '/home/common/matlab/fieldtrip/data/test/original/meg/neuromag122/nmStim150.fif' does not exist

          Error in ==> test_ft_datatype_sens at 50
            sens1 = ft_read_sens(dataset);
      - +@isprivate: "0"
        commentid: "4936"
        comment_count: "1"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-01-25 12:35:12 +0100
        thetext: this probably relates to another neuromag122 dataset being put in the test directory (insteaf of nmStim150) to solve bug 576.
      - +@isprivate: "0"
        commentid: "4988"
        comment_count: "2"
        who:
          +content: j.schoffelen
          +@name: Jan-Mathijs Schoffelen
        bug_when: 2012-01-27 08:47:59 +0100
        thetext: assigned bug to get all bugs lined up before the grand bug binge
      - +@isprivate: "0"
        commentid: "5102"
        comment_count: "3"
        who:
          +content: jorn
          +@name: Jörn M. Horschig
        bug_when: 2012-02-01 17:08:51 +0100
        thetext: "1) The nmStim150.fif file does exist nowhere.\n\n2) There is a jg_single_01.raw.fif instead, but that cannot be opened, although the file exists:\n??? Error using ==> fiff_open at 49\nCannot open file\nH:\\common\\matlab\\fieldtrip\\data\\test\\original\\meg\\neuromag122\\jg_single_01raw.fif\n\nError in ==> fiff_read_meas_info at 82\n    [ fid, tree ] = fiff_open(source);\n\nError in ==> ft_read_header at 1145\n    orig = fiff_read_meas_info(filename);\n\nError in ==> ft_read_sens at 154\n    hdr = ft_read_header(filename, 'headerformat', fileformat);\n\nError in ==> test_ft_datatype_sens at 56\n  sens1 = ft_read_sens(dataset);\n\n\n\n3) The testscript futher fails on \nH:\\common\\matlab\\fieldtrip\\data\\test\\original\\meg\\yokogawa440\\S1_MEG_Epoch.raw\n\nBecause the two sens structures are not similar (assertion fails)"
      - +@isprivate: "0"
        commentid: "5159"
        comment_count: "4"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-02-04 10:55:23 +0100
        thetext: |-
          (In reply to comment #3)

          Regarding the neuromag data: I recall that the nmStim150 dataset was from a 306 channel vectorrview system, and not a 122 channel system. The jg_single_01raw dataset is a much older one from Joachim Gross' neuromag system in Dusseldorf (i.e. their previous system).

          I can confirm the file reading problems, the permissions are incorrect. I fixed them:

          roboos@mentat002> ll
          total 46848
          -rwx------ 1 roboos fieldtrip 47778658 Dec 14 12:24 jg_single_01raw.fif
          roboos@mentat002> chmod 644 jg_single_01raw.fif
      - +@isprivate: "0"
        commentid: "5272"
        comment_count: "5"
        who:
          +content: jorn
          +@name: Jörn M. Horschig
        bug_when: 2012-02-13 16:44:44 +0100
        thetext: |-
          yep, but now assertion fails because sens1 and sens2 are empty for this dataset

          btw, the yokogawa assertion fails because:

          K>> sum(sens2.chanori)

          ans =

               0     0     0


          bbuuuut, should that be my concern? It's drifting off from the initial problem that's filed here
      - +@isprivate: "0"
        commentid: "8477"
        comment_count: "6"
        who:
          +content: jorn
          +@name: Jörn M. Horschig
        bug_when: 2012-11-28 17:58:42 +0100
        thetext: runs smoothly since many commits already
